### PR TITLE
Simplify interface examples

### PR DESCRIPTION
**Pull request by brentyi**
- Renamed demos to examples
- Updated README
- Added a `print_status.py` example
- Removed the waypoint/motion recording demos -- maybe this is too extreme? Maybe these should be moved to `blue_demos`?

@wuphilipp for thoughts

### CODE REVIEW

File: examples/gripper_controller.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+# A basic example of using BlueInterface for gripper control.
+# It allows a user to open and close the gripper.
+
+import sys
```
Comment (wuphilipp) [inline examples/gripper_controller.py]: Do we still need this import?

File: examples/gripper_controller.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from blue_interface import BlueInterface
+import numpy as np
+
+side = "right"
+ip = "127.0.0.1"
+blue = BlueInterface(side, ip)
+
+blue.calibrate_gripper()
+
+opened = True
```
Comment (wuphilipp) [inline examples/gripper_controller.py]: With the new calibrate gripper service, the gripper will be closed after calibration. We should probably that behavior in ```blue_core```.

File: examples/print_status.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+# A basic example of using BlueInterface for joint positions control.
```
Comment (wuphilipp) [inline examples/print_status.py]: This comment should be updated.

**Comment by brentyi** [comment]
Fixed!

**Pull request merged**